test/e2e/manifests/csi-nfs-controller.yaml

```diff
@@ -0,0 +1,108 @@
+---
+kind: Deployment
```

Comment:
why not use yaml config here: https://github.com/kubernetes-csi/csi-driver-nfs/tree/master/deploy/kubernetes

Reply:
That config uses the image pushed to the microsoft container registry. We want a way to locally build an image of the plugin and deploy it on KinD as a part of the CI. So the workflow first builds the image and pushes it to KinD. We needed a way for KinD to pick-up this local image, and hence a separate manifest with a different image name.
I think this is fine for a PoC, but as @msau42 suggested (https://github.com/kubernetes-csi/csi-driver-nfs/pull/69#discussion_r510487906), we can use Kustomize to reduce redundancy.

Reply:
another way is use same way as smb csi driver: set up helm config in testing
https://github.com/kubernetes-csi/csi-driver-smb/blob/e9c2dd3a6ad5d62670177a16cad9b37f4bdabf98/Makefile#L72-L86

is it possible to set up this test for smb csi driver first? @mayankshah1607 by that way, we could find out which way is best.